### PR TITLE
[PET - MLIR] replaced dump with print

### PR DESCRIPTION
**Pull request by Komisarczyk**
replaced a dump with print(outs()) in mlirCodegen.cc

### CODE REVIEW

**Review by chelini — CHANGES_REQUESTED** [review]
Thanks for the patch. Can you please follow the same style we use in the others commit messages? Specifically, add the tag [PET-to-MLIR] to the commit message. The title can be: "Use std::out instead of dump".

File: mlir/test/mlir-pet/test_gemm.mlir [hunk 1]
```diff
@@ -0,0 +1,55 @@
+ // RUN: ./../../../../bin/mlir-pet /llvm-project/mlir/tools/mlir-pet/inputs/gemm.c | FileCheck %s
```
Comment (chelini) [inline mlir/test/mlir-pet/test_gemm.mlir:1]: Can we avoid this? It looks a bit ugly to specify the full path.

File: mlir/test/mlir-pet/test_gemm.mlir [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+// CHECK:  func @scop_entry([[arg2:[%][a-z0-9]*]]: memref<1024x1024xf32>, [[arg3:[%][a-z0-9]*]]: memref<1024x1024xf32>, [[arg4:[%][a-z0-9]*]]: memref<1024x1024xf32>, [[arg5:[%][a-z0-9]*]]: f32, [[arg6:[%][a-z0-9]*]]: f32) {
+// CHECK-NEXT:			      "affine.for"() ( {
+// CHECK-NEXT:    [[b0:[\^][a-z0-9]*]]([[arg0:[%][a-z0-9]*]]: index):	// no predecessors
+// CHECK-NOT:        {{[^ ]+}}
+// CHECK-NEXT:       "affine.for"() ( {
+// CHECK-NOT:        {{[^ ]+}}
```
Comment (chelini) [inline mlir/test/mlir-pet/test_gemm.mlir:7]: What are you trying to check over here?

**Review by chelini — CHANGES_REQUESTED** [review]

File: mlir/test/mlir-pet/test_gemm.mlir (hunk 1 above)
Comment (Komisarczyk) [reply mlir/test/mlir-pet/test_gemm.mlir:1]: It is a relative path and it is awkward to fix as lit cannot cope with an absolute path. For now i dont know how to fix that.

**Review by chelini — APPROVED** [review]

**Pull request merged**